src/pages/docs/experimental-languages/js-language-guide/rpc.mdx

```diff
@@ -1,6 +1,242 @@
 import { Callout } from "nextra/components"
 import { Steps } from "nextra/components"
 
-# Worker to Worker communication for Javascipt components
+# Worker to Worker communication for JavaScript components
 
 See the [Worker to Worker communication page](/docs/concepts/worker-to-worker-communication) for a general overview of how workers can invoke each other in Golem.
+
+## Tooling setup
+
+Make sure that all required tools are installed according to the [setup](/docs/experimental-languags/kss-language-guide/setup) page, including the _Worker to Worker communication_ parts.
```

Comment:
`(/docs/experimental-languags/kss-language-guide/setup)` look like typos in link

Reply:
https://github.com/golemcloud/docs/pull/78/commits/b02500d6f82a48e816d16e163243b2b9a1cfd92f